src/class_smtpServer.py

```diff
@@ -171,15 +189,19 @@ def run(self):
         asyncore.loop(1)
 
 
-def signals(signal, frame):
+def signals(_, __):
```

Comment:
`def signals(*args):` maybe

Reply:
Yes, that's looks good. 
Thank you so much.

Reply:
Checks failed ,pylint error for unused variable. **args**